### PR TITLE
ENH/FIX: Fixes and Improvements

### PR DESCRIPTION
**Pull request by hhslepicka**
**IMPORTANT:** Depends on https://github.com/pcdshub/typhos/pull/229

- [x] Logo
Using the LCLS logo 🤷‍♂ 
- [x] Splash Screen
By having a Splash Screen we can show the user that the system is not locked or doing nothing in case it takes a while to load devices. This was fixed with @ZLLentz 's PR to pcdsdevices but it is good to have that in here in any case.
- [x] Load devices for Main Screen grid in a background thread while users are presented with the Splash Screen.
- [x] Dark Style
Somewhat fixed the Dark Stylesheet and added an option to the launcher to select the dark style: "--dark"
- [x] Typhon -> Typhos
Switched to the new package name
- [x] Fix layout and sizing rules.
Opening an item from the grid can end up hiding part of the grid. We must fix that soon... maybe with this PR as well. This was done with https://github.com/pcdshub/lucid/pull/22/commits/803b93facb6ea5b2ec49e8da886bd299a6f0b137 but it will require some work at the Typhos layer to expand the panel.

### CODE REVIEW

File: lucid/main_window.py [hunk 1]
```diff
@@ -152,8 +155,12 @@ def setup_ui(self):
         # Use the dockmanager for the main window - it will set itself as the
         # central widget
         self.dock_manager = QtAds.CDockManager(self)
-        self.dock_manager.setStyleSheet(
-            open(MODULE_PATH / 'dock_style.css', 'rt').read())
+        if self.dark:
+            self.dock_manager.setStyleSheet(
+                open(MODULE_PATH / 'dock_style_dark.css', 'rt').read())
```
Comment (klauer) [inline lucid/main_window.py]:
```suggestion
                open(MODULE_PATH / ('dock_style_dark.css' if dark else 'dock_style.css'), 'rt').read())
```
🚲 🏠

File: lucid/splash.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import os
+from qtpy import QtGui, QtWidgets, QtCore
+
+import typhos.utils
+
+
+class Splash(QtWidgets.QDialog):
```
Comment (klauer) [inline lucid/splash.py]: 💥

File: requirements.txt [hunk 3]
```diff
@@ -3,5 +3,5 @@ git+https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System
 ophyd
 qtpy
 qtpy
-git+https://github.com/pcdshub/typhon
+typhos
```
Comment (klauer) [inline requirements.txt]: 👏

**Comment by teddyrendahl** [comment]
# [Codecov](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@04f807c`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=tree)

```diff
@@            Coverage Diff            @@
##             master      #22   +/-   ##
=========================================
  Coverage          ?   38.05%           
=========================================
  Files             ?        7           
  Lines             ?      896           
  Branches          ?        0           
=========================================
  Hits              ?      341           
  Misses            ?      555           
  Partials          ?        0           
```


| [Impacted Files](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [lucid/\_\_main\_\_.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvX19tYWluX18ucHk=) | `0.00% <0.00%> (ø)` | |
| [lucid/\_\_init\_\_.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvX19pbml0X18ucHk=) | `100.00% <0.00%> (ø)` | |
| [lucid/utils.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvdXRpbHMucHk=) | `54.21% <0.00%> (ø)` | |
| [lucid/main\_window.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvbWFpbl93aW5kb3cucHk=) | `40.52% <0.00%> (ø)` | |
| [lucid/splash.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvc3BsYXNoLnB5) | `25.00% <0.00%> (ø)` | |
| [lucid/launcher.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvbGF1bmNoZXIucHk=) | `0.00% <0.00%> (ø)` | |
| [lucid/overview.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvb3ZlcnZpZXcucHk=) | `43.57% <0.00%> (ø)` | |

------

[Continue to review full report at Codecov](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=footer). Last update [04f807c...7f5db8c](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by teddyrendahl** [comment]
# [Codecov](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=h1) Report
> :exclamation: No coverage uploaded for pull request base (`master@04f807c`). [Click here to learn what that means](https://docs.codecov.io/docs/error-reference#section-missing-base-commit).
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=tree)

```diff
@@            Coverage Diff            @@
##             master      #22   +/-   ##
=========================================
  Coverage          ?   37.97%           
=========================================
  Files             ?        7           
  Lines             ?      898           
  Branches          ?        0           
=========================================
  Hits              ?      341           
  Misses            ?      557           
  Partials          ?        0           
```


| [Impacted Files](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=tree) | Coverage Δ | |
|---|---|---|
| [lucid/\_\_init\_\_.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvX19pbml0X18ucHk=) | `100.00% <0.00%> (ø)` | |
| [lucid/launcher.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvbGF1bmNoZXIucHk=) | `0.00% <0.00%> (ø)` | |
| [lucid/main\_window.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvbWFpbl93aW5kb3cucHk=) | `40.52% <0.00%> (ø)` | |
| [lucid/overview.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvb3ZlcnZpZXcucHk=) | `43.57% <0.00%> (ø)` | |
| [lucid/\_\_main\_\_.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvX19tYWluX18ucHk=) | `0.00% <0.00%> (ø)` | |
| [lucid/utils.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvdXRpbHMucHk=) | `54.21% <0.00%> (ø)` | |
| [lucid/splash.py](https://codecov.io/gh/pcdshub/lucid/pull/22/diff?src=pr&el=tree#diff-bHVjaWQvc3BsYXNoLnB5) | `25.00% <0.00%> (ø)` | |

------

[Continue to review full report at Codecov](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=footer). Last update [04f807c...5608b42](https://codecov.io/gh/pcdshub/lucid/pull/22?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by hhslepicka** [comment]
@klauer do you want to merge this in and we proceed with another PR or should we keep this PR going?

**Comment by klauer** [comment]
Yes

**Pull request merged**